lib/graphql/schema.graphql

```diff
@@ -80,9 +80,13 @@ enum Gender {
 
 enum ApplicationStatus {
   PENDING
+  INPROGRESS
   APPROVED
   REJECTED
   COMPLETED
+  EXPIRING
+  EXPIRED
+  ACTIVE
```

Comment:
This enum is used for the application processing steps, so it should only contain `PENDING`, `APPROVED`, `REJECTED` and `COMPLETED`.

`EXPIRING`, `EXPIRED` and `ACTIVE` are computed dynamically. `INPROGRESS` is equal to `APPROVED` (if an application is approved but not yet completed, it is currently being processed).